src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/IInstanceEventRepository.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Models;
+
+namespace Altinn.Platform.Storage.Repository
+{
+    /// <summary>
+    /// Interface for interaction with InstanceEvent Repository
+    /// </summary>
+    public interface IInstanceEventRepository
```

Comment:
do we need method to get or delete one event?  /instance-events/{eventId}

Reply:
@dskogan  Based on the functional requirements, no. 
But if we should implement one anyways is another question. I don't really see the need as we are still unclear if it should be possible to delete these events at all..